macos/Classes/DynamicColorPlugin.swift

```diff
@@ -0,0 +1,28 @@
+import Cocoa
+import FlutterMacOS
+
+public class DynamicColorPlugin: NSObject, FlutterPlugin {
+  public static func register(with registrar: FlutterPluginRegistrar) {
+    let channel = FlutterMethodChannel(name: "io.material.plugins/dynamic_color", binaryMessenger: registrar.messenger)
+    let instance = DynamicColorPlugin()
+    registrar.addMethodCallDelegate(instance, channel: channel)
+  }
+
+  public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
+    switch call.method {
+    case "getControlAccentColor":
+      var nsColor = NSColor.systemBlue
+      if #available(macOS 10.14, *) {
```

Comment:
Maybe return null if it Is below 10.14?

Reply:
That's consistent with the Android implementation

Reply:
Done